projects_test.go

```diff
@@ -116,3 +116,50 @@ func TestAccCreateNonDefaultOrgProject(t *testing.T) {
 		t.Fatalf("Expected new project to not be part of org %s", orgPath)
 	}
 }
+
+func TestAccListProjects(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c := setup(t)
+
+	defer projectTeardown(c)
+
+	rs := testProjectPrefix + randString8()
+
+	u, _, err := c.Users.Current()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	orgPath := "/organizations/" + u.DefaultOrganizationID
+
+	pcr := ProjectCreateRequest{Name: rs}
+	p, _, err := c.Projects.Create(&pcr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if p.Organization.URL != orgPath {
+		t.Fatalf("Expected new project to be part of org %s, not %v", orgPath, p.Organization)
+	}
+
+	listOpt := &ListOptions{
+		Includes: "members",
+	}
+	projs, _, err := c.Projects.List(listOpt)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, proj := range projs {
+		if proj.ID == p.ID {
+			if len(proj.Users) == 0 {
```

Comment:
This `if` will never be true, even if you don't specify `"include": ["members"], there will be slice of the "members" (User) struct with `Href` attribute populated, just the other attributes of the User struct won't be populated.
It doesn't do any hurt here, but it doesn't do any good either. Up to you if you drop it.